concepts/string-interpolation/links.json

```
@@ -1,2 +1,10 @@
-{
-}
\ No newline at end of file
+[
+  {
+    "url": "https://haxe.org/manual/std-String.html",
+    "description": "string"
+  },
+  {
+    "url": "https://haxe.org/manual/lf-string-interpolation.html",
+    "description": "string-interpolation"
```

```suggestion
    "description": "String interpolation"
```

See https://github.com/exercism/haxe/pull/154#discussion_r583473891

---

Thanks!